paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -17,33 +22,87 @@ import javax.inject.Inject
 @CustomerSessionScope
 @Suppress("unused")
 internal class CustomerSheetViewModel @Inject constructor(
+    application: Application,
     private val configuration: CustomerSheet.Configuration,
     private val customerAdapter: CustomerAdapter,
     private val lpmRepository: LpmRepository,
-) : ViewModel() {
+) : AndroidViewModel(application) {
```

Comment:
We should keep this as `ViewModel` if we can. Since `application` is only used to fetch a payment method’s name, can you extract this into a `PaymentMethodNameProvider` interface (or similar)?